apps/web/src/utils/authentication/store.ts

```diff
@@ -0,0 +1,17 @@
+import type { UserProfile } from '@walless/core';
+import { modules } from '@walless/ioc';
+import type { SettingDocument } from '@walless/store';
+
+export const setProfile = async (profile: UserProfile) => {
```

Comment:
I think we already have this `setProfile` somewhere in `utils/state` -- this seems a duplication

Reply:
I moved that method to this place